### ISSUE TITLE
Warning: Failed prop type: The prop `"aria-label"` is marked as required in `Grid`, but its value is `undefined`.

### ISSUE THREAD

**Issue by rpajunen** [frontend-collective/react-sortable-tree#858]
Same as this issue #https://github.com/bvaughn/react-virtualized/issues/1577?

**Comment by allancmm** [comment]
I get the same error
